src/java.desktop/unix/native/common/awt/CUPSfuncs.c

```diff
@@ -246,6 +246,7 @@ Java_sun_print_CUPSPrinter_getCupsDefaultPrinters(JNIEnv *env,
     for (i = 0; i < num_dests; i++) {
             utf_str = JNU_NewStringPlatform(env, dests[i].name);
             if (utf_str == NULL) {
+                (*env)->ExceptionClear(env);
```

Comment:
Shouldn't we also need to do same in https://github.com/openjdk/jdk/blob/master/src/java.desktop/unix/native/common/awt/CUPSfuncs.c#L357, L367, L381 since we do at L348

Reply:
Nothing changed there, there's no complaint about that.
Perhaps the tool thinks it is OK to make the call to throw another exception ?
I'll ask about that.
FWIW if I go making changes to these places where there is no complaint I don't think I'd automatically clear
an exception in order to throw a different one .. I'd prefer to propagate the original.